### ISSUE TITLE
Unable to drag response tab box larger

### ISSUE THREAD

**Issue by pr0letariat** [zaproxy/zaproxy#3961]
This is a recent issue I've come across.

I'm unable to drag the `Response` tab box across. I can drag it by a tiny amount then it does not drag any more.

![drag-issue](https://user-images.githubusercontent.com/14975253/31691502-37c0319c-b38e-11e7-9042-7308a39072a1.gif)

Nothing in `zap.log`.

Running `Version 2.6.0`

**Comment by thc202** [comment]
That seems to be caused by the Quick Start tab. Which version of Quick Start add-on are you using? Latest?

**Comment by pr0letariat** [comment]
Yes, when I check for updates, it doesn't say it needs updating.

**Comment by pr0letariat** [comment]
Anything I can do to fix this? Would say this is a pretty big issue as prevents me from seeing the full response in a nice, readable format.

**Comment by thc202** [comment]
Right, could you share a screen shot with both (whole) tabs Quick Start and Response? Which other tabs do you have with the Quick Start? (Just Request?)

**Comment by psiinon** [comment]
As a temp fix you could try just closing the Quick Start tab :)

**Comment by thc202** [comment]
Yeah, was going to suggest trying that (which really helps to know which tab is causing the problem).

**Comment by pr0letariat** [comment]
I tried closing the Quick Start tab but that didn't resolve it. I don't have any other tabs except Quick Start, Request and Requester (with Response on the right). Not sure what you mean by both the whole tabs in the screenshot?

**Comment by thc202** [comment]
Did you try closing the Requester tab? (That might have enough UI components to prevent the minimisation.) Regarding the screen shot with the whole tabs, was to see them all in the same image (to check which components were or not visible).

**Comment by pr0letariat** [comment]
Yep, closing Requester solves the issue.

**Comment by thc202** [comment]
Cool, issue updated. (Maybe the Requester tab should have a scroll pane?)

**Comment by thc202** [comment]
Updated the title to reflect the actual problem (which includes other issues that are caused by that).

**Comment by ArkaprabhaChakraborty** [comment]
Also, I would have liked a double bar instead of a single bar with all options... probably it would also help when I'm "compressing" the tab :)

**Comment by thc202** [comment]
(This issue also applies to the Manual Request Editor which uses the same panels.)

**Comment by ArkaprabhaChakraborty** [comment]
Oh, not in the manual editor tho... There the one single bar looks much better... The requestor on the other hand requires a resizing more often so it might help..

**Comment by thc202** [comment]
The code is the same, changing one will change the other though.

**Comment by ArkaprabhaChakraborty** [comment]
That... sounds like a problem :)
